schema/data/prod/json_schema/milestone.json

```diff
@@ -51,6 +51,11 @@
             "type": "string",
             "title": "Certified Professional Accountant",
             "enum": ["Certified Professional Accountant"]
+          },
+          {
+            "type": "string",
+            "title": "N/A",
```

Comment:
Maybe it should say "Select a Designation" instead of "N/A"? I think all the other dropdowns say "Select a XX" if blank

Reply:
We want to allow users to select a blank option if they don't have a professional designation. An option could simply be "None" or "No Professional Designation"